CablePairTester_CommonCfg.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * CablePairTester_CommonCfg.h
+ *
+ *  Created on: Feb 22, 2021
+ *      Author: PUL1KOR
+ */
+
+#ifndef CABLEPAIRTESTER_COMMONCFG_H_
+#define CABLEPAIRTESTER_COMMONCFG_H_
+
+#include <stdio.h>
+#include <assert.h>
+
+/************
+ * Colour Pair as per standard
+ ************/
+
+enum MajorColor_ten {WHITE, RED, BLACK, YELLOW, VIOLET};

```

Comment:
What does the _ten stand for in the name? The number of major colors is not ten...